docs/standard/serialization/system-text-json/customize-properties.md

```diff
@@ -228,7 +233,108 @@ By default, properties are serialized in the order in which they're defined in t
 
 :::code language="csharp" source="snippets/how-to-6-0/csharp/PropertyOrder.cs":::
 
+## Use Github Copilot to customize property names and order 
+
+You can use GitHub Copilot in your IDE to generate code to customzie names and order of serialized properties.
+
+If you're using [Visual Studio 2022 version 17.8 or later](/visualstudio/releases/2022/release-notes), you can try the AI-driven [GitHub Copilot in Visual Studio](/visualstudio/ide/visual-studio-github-copilot-install-and-states) to generate code that uses `System.Text.Json` to customize property names and order in the JSON output from serializtion. Submit your question as a prompt in the Copilot chat window, as in the following example. You can also submit prompts using [inline chat](/visualstudio/ide/visual-studio-github-copilot-chat#ask-questions-in-the-inline-chat-view) in the editor window itself.
+
+> [!NOTE]
+> GitHub Copilot is powered by AI, so surprises and mistakes are possible. Make sure to verify any generated code or suggestions. For more information about the general use of GitHub Copilot, product impact, human oversight, and privacy, see [GitHub Copilot FAQs](https://github.com/features/copilot#faq).
+
+The following example shows you how to use Copilot to modify existing code to customize property names and order when serializing to JSON.
+
+1. Add the following C# example code to a code file `Example.cs` in your editor.
+   In Visual Studio, you can use a C# console application project to try this example.
+   
+   :::code language="csharp" source="snippets/how-to-6-0/csharp/copilot-example.cs":::
+
+  `Example.cs` code does the following:
+  - Creates an instance of the `Person` class and initializes its properties with values.
+  - Serializes the `person` object to a JSON string using `JsonSerializer.Serialize`.
+  - Prints the following JSON string to the console:
+    ```json
+    {"FirstName":"John","LastName":"Doe","Age":30,"Country":"USA"}
+    ```
+
+1. In Copilot Chat, enter the following prompt to modify the code to customize names and order of the JSON serialization output.
```

Comment:
Something's off with the indentation, because the numbering restarts at 1 here in the preview.

Reply:
Thought I'd fixed it. Trying again, seems to be fixed now.